package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.10.0",
     "@emotion/styled": "^11.10.0",
+    "@mui/material": "^5.10.0",
```

Comment:
Do you really need this one?

Reply:
I will replace notistack with different snackbar library to exclude mui dependency